kafka-connect/modules/kafka-connect/added/scripts/kafka_connect_config_generator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This file is changed one more time. Get the updated version from `master` branch of the `strimzi-kafka-operator` project.